### ISSUE TITLE
bug(a11y): The docs/landing page has accessibility issues

### ISSUE THREAD

**Issue by phacks** [theodo/falco#79]
Following an accessibility audit by @AlbericTrancart, here are the problems that need solving on https://getfal.co:

## Images

- [x] The company logos all have the same, “BNP Paribas” `alt` (#83)
![image](https://user-images.githubusercontent.com/2587348/67852545-e3b54d80-fb0c-11e9-8d18-0695dcac7adc.png)
- [x] Decorative images should have an empty `alt` (#83)
![image](https://user-images.githubusercontent.com/2587348/67852577-f3cd2d00-fb0c-11e9-9562-4852f9f38967.png)

## Content

- [ ] The HTML is invalid according to the W3C validator ([link to audit](https://validator.w3.org/nu/?doc=https%3A%2F%2Fgetfal.co%2F)): 14 errors, 5 warnings
- [x] There is a jump from `h1` to `h3` without any `h2` tags (#83)

## CSS

- [ ] The responsive sidebar should be masked via `display:none` or `aria-hidden`

## Navigation

- [x] The “Read the docs” and “Deploy to Heroku” buttons do not have any `focus` style (#83)
- [ ] There is no `skip to content` link

**Comment by phacks** [comment]
@all-contributors please add @AlbericTrancart for ideas

**Comment by phacks** [comment]
We went over the remaining items with @AlbericTrancart:

> The responsive sidebar should be masked via display:none or aria-hidden

The issue does not appear on desktop—on mobile, the links inside the menu are in fact accessible.

> The HTML is invalid according to the W3C validator (link to audit): 14 errors, 5 warnings

The issues (usually invalid HTML properties) come from Docusaurus. I’ll open an issue / a PR in the Docusaurus V2 repo.

> There is no skip to content link

Not very relevant in the docs pages as the content can be reached fairly easily with the `See the docs` link.

I’m closing the issue for now—if anybody notices other a11y issues, please reopen it!
